src/main/java/gregtech/common/pipelike/cable/BlockCable.java

```diff
@@ -105,10 +106,11 @@ public int getActiveNodeConnections(IBlockAccess world, BlockPos nodePos, IPipeT
     @Override
     public void onEntityCollidedWithBlock(World worldIn, BlockPos pos, IBlockState state, Entity entityIn) {
         Insulation insulation = getPipeTileEntity(worldIn, pos).getPipeType();
+        boolean damageOnLossless = ConfigHolder.doesLosslessDamage;
         if (!worldIn.isRemote && insulation.insulationLevel == -1 && entityIn instanceof EntityLivingBase) {
             EntityLivingBase entityLiving = (EntityLivingBase) entityIn;
             EnergyNet energyNet = getWorldPipeNet(worldIn).getNetFromPos(pos);
-            if (energyNet != null && energyNet.getAllNodes().get(pos).data.lossPerBlock > 0) {
+            if (energyNet != null && (energyNet.getAllNodes().get(pos).data.lossPerBlock > 0 || damageOnLossless)) {
```

Comment:
I think we should use this instead for quicker evaluation in case that `damageOnLossless == true`.
```suggestion
            if (energyNet != null && (damageOnLossless || energyNet.getAllNodes().get(pos).data.lossPerBlock > 0 )) {
```